common.blocks/vow/vow.ru.md

```diff
@@ -0,0 +1,112 @@
+
+# vow
+
+Блок служит для подключения библиотеки [Vow](https://github.com/dfilatov/vow). 
+
+Vow – библиотека для работы с промисами, реализующая стандарт [Promises/A+](http://promisesaplus.com/) и поддержку спецификации [ES6 Promises](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-promise-objects).
+
+Блок `vow` реализован в технологии `vanila.js` и подходит для использования как на клиенте, так и на сервере.
+
+## Использование
+
+### Создание промиса
+Есть два возможных пути создания промиса:
+
+#### 1. С помощью deferred
+
+```js
+function doSomethingAsync() {
+    var deferred = vow.defer();
+    
+    // теперь можно воспользоваться методами resolve, reject и notify 
+    // для изменения состояний `deferred`. 
+    // Например, `defered.resolve('ok');`
+        
+    return deferred.promise(); // затем возвращаем promise, чтобы предоставить возможность подписки
+}
+
+doSomethingAsync().then(
+    function() {}, // реакция на успешное разрешение (resolve)
+    function() {}, // реакция на разрешение отказом (reject)
+    function() {}  // реакция на изменение состояния (notify)
+);
+```
+
+
+Разница между `deferred` и `promise` состоит в том, что, `deferred` имеет методы для изменения состояний промиса – разрешения, отказа и успеха, тогда как промис позволяет лишь подписаться на эти состояния.
+
+Создадим блок, выполняющий некоторое асинхронное действие. Для наглядности воспользуемся методом `setTimeout` со случайной задержкой: 
+
+```js
+modules.require(['vow'], function(vow) {
+    var a = [1, 2, 3, 4, 5], 
+    // массив значений
+        _doAsync = function(val){
+            // создаем случайную задержку в диапазоне от 0 до 3 сек
+            var delay = (Math.round(Math.random()*3000)), 
+                defered = vow.defer();
+
+            // вызываем setTimeout со случайной задержкой и вызываем defered.resolve по ее истечению
+            setTimeout(function() { defered.resolve(val + ' delay was ' + delay) }, delay);
+            
+            // возвращаем промис, чтобы предоставить возможность подписки
+            return defered.promise();
+        };
+
+    for(i = 0; i < a.length; i = i + 1){

```

Comment:
> in theory all commited files should be valid because of jscs on pre-commit hook and in travis ;)

Yep. I'm just talking about Сливки собрать into one simple file with all rules usage examples.